src/components/application_manager/src/commands/mobile/send_location_request.cc

```diff
@@ -62,8 +62,7 @@ void SendLocationRequest::Run() {
 
   smart_objects::SmartObject& msg_params = (*message_)[strings::msg_params];
   if (msg_params.keyExists(strings::delivery_mode)) {
-    const std::vector<std::string>& allowed_params =
-        parameters_permissions().allowed_params;
+    const RPCParams& allowed_params = parameters_permissions().allowed_params;
     if (allowed_params.end() == std::find(allowed_params.begin(),
                                           allowed_params.end(),
```

Comment:
@LuxoftAKutsan There is in_range<>() in helpers.h. I think instead find(), == end() etc. can be more clear and shorter.